app/src/utils/utils.ts

```diff
@@ -121,3 +118,19 @@ export function displayResults(
   const end = Math.min(page * perPage, numFound);
   return `${start}-${end} of ${numFound}`;
 }
+
+export function formatHighlightText(highlights) {
```

Comment:
There are really good tests to test this but they're indirect. For a utility function like this, add more direct function tests.